README.md

```diff
@@ -367,6 +367,7 @@ abstract class Credential
 class PrivateKeyCredential : Credential
 {
   PrivateKeyCredential(string path);
+  PrivateKeyCredential(string path, string? passphrase)
```

Comment:
You can merge this with the existing overload, and initialize `= null`.

I think it's ok to do a minor breaking change like this at this point to reduce the number of overloads a user needs to work with.